### ISSUE TITLE
Select & SelectMultiple - form with no placeholder accessibility

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7211]
### Expected Behavior
If the Select or SelectMultiple is in a FormField with a label and no placeholder text the label should be read out by a screenreader when the Select or SelectMultiple receives focus.

### Actual Behavior
The label is not read out when the Select/SelectMultiple receives focus

### Steps to Reproduce

Create a Select component in a FormField with a label and no placeholder text. Try using a screenreader

### Your Environment
Tested with voiceover on Mac

**Comment by Hriishikeshh** [comment]
hey would like to work on this
assign it to me if suited

**Comment by nishantdecode** [comment]
Hey @jcfilben! I have finished working on this issue, but encountering DCO error not sure what to do, any feedback would be helpful, thank you!

**Comment by britt6612** [comment]
@nishantdecode 
You will need the -s flag and a remediation commit. See https://github.com/grommet/grommet/pull/7357/checks?check_run_id=31375867254

looks like you got it on the PR

**Comment by nishantdecode** [comment]
@britt6612 
Thank you for the input. I have corrected the error by using the -s flag. Please take another look at the PR.

**Comment by jcfilben** [comment]
Wave indicates that the label is correctly linked, it may just be an issue with voiceover need to do a little more research to confirm.
